src/core/main.c

```diff
@@ -3482,7 +3482,8 @@ int main(int argc, char *argv[]) {
                                               ANSI_HIGHLIGHT_RED "!!!!!!" ANSI_NORMAL,
                                               "%s.", error_message);
                 freeze_or_exit_or_reboot();
-        }
+        } else if (error_message)
```

Comment:
Doesn't this cause double logging? See e.g line 3468, we just logged `error_message` there…